src/endpoints/test.ts

```diff
@@ -0,0 +1,43 @@
+import { config } from 'dotenv';
+import { join } from 'path';
+import Client from '../index';
+
+config({ path: join(__dirname, '.env.e2e') });
+
+// Extract and validate required environment variables
+const token = 'jKD7wagy7xxHAw.atlasv1.OgOqpRWWNPU5hlzeNvbHXLWnFkxMMFceW9m5LYoyUDzyYoSytwfphRk4V5yu6CYNFCA'
```

Comment:
If you ever do that again I'll drive straight to Marseille and throw you in the see from the Bonne Mère !